cmd/kubeapps-apis/server/server.go

```diff
@@ -95,9 +96,23 @@ func Serve(serveOpts ServeOptions) {
 	// at https://github.com/soheilhy/cmux/issues/64
 	mux := cmux.New(lis)
 	grpcLis := mux.MatchWithWriters(cmux.HTTP2MatchHeaderFieldSendSettings("content-type", "application/grpc"))
+	grpcwebLis := mux.MatchWithWriters(cmux.HTTP2MatchHeaderFieldSendSettings("content-type", "application/grpc-web"))
 	httpLis := mux.Match(cmux.Any())
 
+	webrpcProxy := grpcweb.WrapServer(grpcSrv,
+		grpcweb.WithOriginFunc(func(origin string) bool { return true }),
+		grpcweb.WithWebsockets(true),
+		grpcweb.WithWebsocketOriginFunc(func(req *http.Request) bool { return true }),
+	)
+
+	httpSrv.Handler = http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if webrpcProxy.IsGrpcWebRequest(r) || webrpcProxy.IsAcceptableGrpcCorsRequest(r) || webrpcProxy.IsGrpcWebSocketRequest(r) {
+			webrpcProxy.ServeHTTP(w, r)
+		}
+	})
```

Comment:
I'm confused by this line: it appears to *overwrite* the existing `httpSrv.Handler` (which is the gateway mux), with a handler that handles the grpc-web requests, but doesn't fall back to the existing gateway mux for other requests? Maybe it was forgotten, or maybe I'm missing something?